hota/Mods/HotA banks/Content/config/banks/banks/spit.json

```diff
@@ -13,35 +13,32 @@
 						"animation" : "banks/map/banks/avbone02.def", 
 						"visitableFrom" : [ "---", "-+-", "+++" ], 
 						"mask" : [ "VVVV", "VVBB", "VBAB"],
-						"allowedTerrains":["grass", "swamp"] 							
+						"allowedTerrains":["grass", "swamp"]
```

Comment:
According to wiki: https://heroes.thelazy.net//index.php/Spit only in swamp terrain should be placed. And stack placement is diffent, but I guess it's not implemented in vcmi.

Reply:
Stack placement - yeah, trivial to add though.
I've made separate item about that in VCMI-HotA project.
Same issue with churchyard bank.